bip-0021.mediawiki

```diff
@@ -84,8 +97,7 @@ Also, very likely, what he will find are mostly technical specifications - not t
 ==Forward compatibility==
 Variables which are prefixed with a req- are considered required.  If a client does not implement any variables which are prefixed with req-, it MUST consider the entire URI invalid.  Any other variables which are not implemented, but which are not prefixed with a req-, can be safely ignored.
 
-==Backward compatibility==
```

Comment:
It seems to me that this update should have its own Backward Compatibility section.